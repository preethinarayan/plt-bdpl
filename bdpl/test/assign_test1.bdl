struct small_byte
{
	byte b (fieldsize 4);
}sb;	

struct aa
{
	type struct small_byte[*] asb;
} a (fieldsize 20);

type struct aa bb;

file "x:/test/file_test.dat" inp_file;

read(inp_file,a);
print(a);
bb=a;
print(bb);

file "x:/test/file_test.dat" inp_file;
read(inp_file,a);
