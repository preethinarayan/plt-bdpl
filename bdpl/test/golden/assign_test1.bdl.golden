struct:aa
{
	asb => ARRAY of struct:small_byte
	[
		0 => struct:small_byte
		{
			b => 0x0,
		}
		,
		1 => struct:small_byte
		{
			b => 0xd,
		}
		,
		2 => struct:small_byte
		{
			b => 0x4,
		}
		,
		3 => struct:small_byte
		{
			b => 0x1,
		}
		,
		4 => struct:small_byte
		{
			b => 0x4,
		}
		,
	]
	,
}
struct:aa
{
	asb => ARRAY of struct:small_byte
	[
		0 => struct:small_byte
		{
			b => 0x0,
		}
		,
		1 => struct:small_byte
		{
			b => 0x0,
		}
		,
		2 => struct:small_byte
		{
			b => 0x0,
		}
		,
		3 => struct:small_byte
		{
			b => 0xd,
		}
		,
		4 => struct:small_byte
		{
			b => 0x0,
		}
		,
		5 => struct:small_byte
		{
			b => 0x4,
		}
		,
		6 => struct:small_byte
		{
			b => 0x0,
		}
		,
		7 => struct:small_byte
		{
			b => 0x1,
		}
		,
		8 => struct:small_byte
		{
			b => 0x0,
		}
		,
		9 => struct:small_byte
		{
			b => 0x4,
		}
		,
	]
	,
}
